src/nxdoc/nuxeo-server/installation/nuxeoctl-and-control-panel-usage.md

```diff
@@ -670,8 +680,6 @@ If uninstalling a package by its ID and other versions of the same package are a
 </table>
 </div>
 
-The following commands are always executed in console/headless mode (no GUI): `configure`, `mp-purge`, `mp-add`, `mp-install`, `mp-uninstall`, `mp-request`, `mp-remove`, `mp-hotfix`, `mp-upgrade`, `mp-reset`, `mp-list`, `mp-listall`, `mp-update`, `status`, `showconf`, `mp-show`, `mp-set`, `config`, `encrypt`, `decrypt`, `help`.
```

Comment:
Why deleting this?

Reply:
Because we're not in the ZIP context so GUI makes no sense.